src/Tools/dotnet-watch/src/Internal/MsBuildFileSetFactory.cs

```diff
@@ -74,7 +78,8 @@ public async Task<IFileSet> CreateAsync(CancellationToken cancellationToken)
                             "msbuild",
                             "/nologo",
                             _projectFile,
-                            $"/p:_DotNetWatchListFile={watchList}"
+                            $"/p:_DotNetWatchListFile={watchList}",
+                            _dotNetWatchOptions.SuppressHandlingStaticContentFiles ? "/p:DotNetWatchContentFiles=false" : "",
```

Comment:
Seem odd to have the env var (SUPPRESS) default to false while the property passed in default to true.

Reply:
DotNetWatch has precedence for using BLAH_SUPPRESS_FEATURE pattern to turn off features: https://docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-5.0#dotnet-watch-configuration. MSBuild does not typically do this. You usually set a property to false to turn it off. We want both ways to be work (turn off file watching for content files on a per project basis and turn it off as a watch feature).

I'm open to suggestions though.